ring/ring.go

```diff
@@ -78,6 +78,12 @@ type ReadRing interface {
 
 	// GetTokenRangesForInstance returns the token ranges owned by an instance in the ring
 	GetTokenRangesForInstance(instanceID string) (TokenRanges, error)
+
+	// InstancesInZoneCount returns the number of instances in the ring that are registered in given zone.
+	InstancesInZoneCount(zone string) int
+
+	// ZonesCount returns the number of zones for which there's at least 1 instance registered in the ring.
+	ZonesCount() int
```

Comment:
Do we also need `InstancesCount` to fulfill the `RingCount` [interface](https://github.com/grafana/mimir/blob/16877ec44bc52d4d2f4459ee544dad0e31c8e1f5/pkg/ingester/limiter.go#L18-L22)?

Reply:
it [exists](https://github.com/grafana/dskit/pull/494/files#diff-b51c6d6dfebcf2757957f711c664d9579e1789505a8e771edbe04ede79ff4c9fR59)